vignettes/hys_law.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Hy's Law Implementation Guide"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Hy's Law Implementation Guide}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiral)
+library(admiraldev, warn.conflicts = FALSE)
+library(dplyr, warn.conflicts = FALSE)
+library(tidyr, warn.conflicts = FALSE)
+library(purrr)
+library(admiral.test)
+library(lubridate)
+library(stringr)
+```
+
+# Context
+
+During the drug development process, clinical trials are often required to assess for the potential that the experimental drug can cause severe liver injury, known as a drug induced liver injury (DILI). There are multiple criteria that need to be evaluated to determine and classify a DILI "event". A common rule of thumb for this is, known as Hy's Law, is usually comprised of three parts:
+
+- Elevated alanine aminotransaminase (ALT) or aspartate aminotransaminase (AST) by 3-fold or greater of the upper limit of normal.
+- Elevated serum total bilirubin (BILI) by 2-fold or greater within a window of time, ~14 days after the elevated AST/ALT event.
+- No other reason to explain these increased lab values like preexisting liver disease.
+
+
+# Example
+
+We can use a mixture of `admiral` and basic `tidyverse` functions to assist us in creating an ADLBHY dataset. In the walkthrough below we will use the adlb dataset created from the call `use_ad_template("adlb")`. Due to the size of the dataset, we only included the following USUBJID's: 01-701-1015, 01-701-1023, 01-701-1028, 01-701-1033, 01-701-1034, 01-701-1047, 01-701-1097, 01-705-1186, 01-705-1292, 01-705-1310, 01-708-1286. 
```

Comment:
```suggestion
We can use a mixture of `{admiral}` and basic `{tidyverse}` functions to assist us in creating an ADLBHY dataset. In the walkthrough below we will use the ADLB dataset created from the call `use_ad_template("adlb")`. Due to the size of the dataset, we only included the following USUBJID's: 

* 01-701-1015, 01-701-1023, 01-701-1028, 01-701-1033, 01-701-1034, 01-701-1047, 01-701-1097, 01-705-1186, 01-705-1292, 01-705-1310, 01-708-1286. 
```

Reply:
Wondering if the Subjects on a new line will look cleaner?

Reply:
how is the encased-in-ticks look?

![image](https://user-images.githubusercontent.com/43018762/219470582-0b7898f8-1f3a-443b-9532-501813867c1f.png)